HW4/src/isSmaller.js

```diff
@@ -0,0 +1,4 @@
+function isSmaller(a, b) {
+    return a < b ? true : false;
```

Comment:
you can remove trinariy expression and have the same result

Reply:
+